tests/conftest.py

```diff
@@ -51,6 +51,7 @@ def valid_config(monkeypatch, loggedin_status):
                 runtime_key="FUNCTION_KEY",
                 cdf_base_url="https://api.cognitedata.com",
             ),
+            secret="eyJrZXkiOiJ2YWx1ZSJ9Cg==",
```

Comment:
`# this is a mock example of an invalid key!`

Reply:
it's a valid secret which can be decoded and parsed by json.

Reply:
> `# this is a mock example of an val key!`